task/01-strings-tasks.js

```diff
@@ -22,7 +22,7 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-    throw new Error('Not implemented');
+    return value1+value2
```

Comment:
space around operator
Please, try to follow these rules https://javascript.info/coding-style